src/app/content/components/SidebarControl.tsx

```diff
@@ -80,17 +109,27 @@ const connector = connect(
 );
 
 // tslint:disable-next-line:variable-name
-const lockControlState = (isOpen: boolean, Control: React.ComponentType<InnerProps>) =>
-  connector((props: MiddleProps) => <Control
+const lockControlState = (Control: React.ComponentType<InnerProps>, forcedIsOpen: boolean | null = null, ) =>
+  connector((props: MiddleProps) => {
+  const isToCOpened = forcedIsOpen !== null ? forcedIsOpen : props.isOpen;
+
+  return <Control
     {...props}
     data-testid='toc-button'
-    message={isOpen ? openMessage : closedMessage}
-    data-analytics-label={isOpen ? 'Click to close the Table of Contents' : 'Click to open the Table of Contents'}
-    onClick={isOpen ? props.closeToc : props.openToc}
-  />);
+    message={isToCOpened === false ? closedMessage : openMessage}
+    data-analytics-label={isToCOpened === false
+      ? 'Click to open the Table of Contents'
+      : 'Click to close the Table of Contents'}
```

Comment:
These should be internationalized

Reply:
@bethshook Do we really want to internationalize the `data-analytics-label`? In a lot of places it is hard coded

Reply:
Whoops, yeah, I misread what this was